repos.d/deb/apertis.yaml

```diff
@@ -29,6 +29,15 @@
   repolinks:
     - desc: Apertis home
       url: https://www.apertis.org/
+    - desc: Apertis GitLab
+      url: https://gitlab.apertis.org/
+    - desc: Apertis Quality Assurance reports
+      url: https://qa.apertis.org/
+  packagelinks:
+    - type: PACKAGE_SOURCES
+      url: 'https://gitlab.apertis.org/pkg/{srcname}'
```

Comment:
This probably needs to be tied to a branch, e.g. something like `https://gitlab.apertis.org/pkg/{srcname}/-/tree/apertis/{{v}}`

Reply:
Thanks, fixed.